pizzabot.js

```diff
@@ -0,0 +1,119 @@
+//Iteration 1 - Setting everything up.
+const vegetarian = "vegetarian"
+const hawaiian = "hawaiian"
+const pepperoni = "pepperoni"
+const calzone = "calzone"
+const kebab = "kebab"
+
+// Iteration 4 - Finalize the order
+const calculateTotalCost = (event) => {
+    event.preventDefault();
+    //Get nr of pizzas
+    let orderQuantity = document.getElementById("nrOfPizzas").value;
+    //Get pizza sort
+    let orderName = document.getElementById("pizzaSort").value;
+    let pizzaPrice = pricePerPizza(orderName);
+    //Calculate totalcost for pizza
+    const orderTotal = orderQuantity * pizzaPrice
+    //Print out message to customer
+    let orderTime = calculateCookingTime(orderQuantity)
+    let pizzaImg = pizzaImages(orderName)
+    document.getElementById("pizzaImg").src = pizzaImg;
+    let info = `Great, I'll get started on your ${orderName} right away, it will cost ${orderTotal} kr. The pizza(s) will take ${orderTime} minutes.`;
+    //Set pizza picture to display block
+    let x = document.getElementById("pizzaImg");
+    x.style.display = "block";
+    //Display customer order confirmation
+    document.getElementById("submitInfo").innerHTML = info;
+
+};
+
+const form = document.getElementById("form");
+form.onsubmit = calculateTotalCost;
```

Comment:
It seems like the `calculateTotalCost` function does a bit more than that 😄 
Maybe we could find another name for it? 
In this case, `handleOnSubmit` would be just fine

Reply:
Good suggestion! Fixed. Changed calculateTotalCost to handleOnSubmit.